src/prefabs/structures/openPage/options/index.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  toggle,
+  variable,
+  option,
+  endpoint,
+  showIf,
+  icon,
+  hideIf,
+  sizes,
+} from '@betty-blocks/component-sdk';
+import { advanced } from './advanced';
+import { tooltip } from './tooltip';
```

Comment:
this import is currently not passing through our Linter. This needs to be changed so it is accepted